vignettes_src/windows.Rmd

```diff
@@ -0,0 +1,245 @@
+
+```
+getwd()
+```
+
+Interpreting this depends on your platform (the machine you are typing commands into):
+
+* **Windows**: A drive like `C:` or `D:` will be local. You should recognise the drive letter as one like `Q:` that was mapped by default as your home or `M:` that you mapped yourself as a project share.
+* **macOS**: The path will likely be below `/Volumes` (but not one that corresponds to a USB stick of course!)
+* **Linux**: The path should start at one of the mount points you have configured in `/etc/fstab` as a network path.
+
+## Project shares and home directories
+
+In general we **strongly recommend** that you use project shares for any serious work, rather than your home directory.  To organise these you should talk to your PI (or if you are a PI, talk to Chris).  The advantages of the project shares is that they are larger (so you will run out of disk space more slowly) and faster than the home shares.  If you launch many tasks at once that use your home share you can get unexpected failures as the disk can't keep up with amount of data being read and written.  Don't use your home directory (generally `Q:` on windows) for anything more intensive than casual experimentation.  We have seen many people with jobs that fail mysteriously on their network home directory when launched in parallel, and this is generally fixed by using a project share.
+
+If you don't know if you have a project share, talk to your PI about if one exists. If you are a PI, talk to Chris (and/or Wes) about getting one set up.
+
+## Mapping network drives
+
+For all operating systems, if you are on the wireless network you
+will need to connect to the department network using ZScaler; see the [ICT documentation](https://www.imperial.ac.uk/admin-services/ict/self-service/connect-communicate/remote-access/unified-access/) for details. If you can get on a wired network you'll likely have a better time because the VPN and wireless network seems less stable in general (it is not clear how this will be in the new building at White City, or indeed how many wired points there will be).
+
+Below, instructions for setting up depend on the sort of computer you are typing commands into at the moment (not the cluster type).
+
+### Windows
+
+If you are using a windows machine in the DIDE domain, then
+your network drives are likely already mapped for you.  In fact you
+should not even need to map drives as fully qualified network names
+(e.g. `//fi--didef3/tmp`) should work for you.
+
+### macOS
+
+In Finder, go to `Go -> Connect to Server...` or press `Command-K`.
+In the address field write the name of the share you want to
+connect to.  Useful ones are
+
+* `smb://fi--san03.dide.ic.ac.uk/homes/<username>` -- your home share
+* `smb://fi--didef3.dide.ic.ac.uk/tmp` -- the temporary share
+
+At some point in the process you should get prompted for your
+username and password, but I can't remember what that looks like.
```

Comment:
TO-DO - we should improve that last sentence a bit. 
Maybe also we should avoid talking about the temp drive, or mention that it might be terrible.